STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/web/repository/DeploymentController.java

```diff
@@ -393,6 +395,26 @@ public String openSelectedProjects() {
         return null;
     }
 
+    private RulesProject getRulesProject(UserWorkspace workspace,
+            DeploymentDescriptorItem deployment) throws ProjectException {
+        try {
+            return workspace.getProject(deployment.getRepositoryId(), deployment.getName(), false);
+        } catch (ProjectException e) {
+            Repository repo = workspace.getDesignTimeRepository().getRepository(deployment.getRepositoryId());
+            if (repo.supports().mappedFolders()) {
+                String mappedName = ((MappedRepository) repo).getMappedName(deployment.getName(), deployment.getPath());

```

Comment:
Same as repositoryId, deployment.getPath() can be null if Deploy Configuration was created in previous version of WebStudio.
If it's null we can skip further steps.

Reply:
fixed